### PR TITLE
Split key-value line on first colon

### LINKED ISSUE #242: IPv6 address in redis output breaks parsing
When there's an IPv6 address in the output of Redis, the line is skipped, resulting in missing metrics.

For example:
```
slave0:ip=2a00:1450:400e:808::200e,port=6379,state=online,offset=9694740951,lag=1
```
This line will never be handled.

Line of code causing this:
https://github.com/oliver006/redis_exporter/blob/e3f43cfe305f8d20d931de6eb84e32c664b7edd1/exporter/redis.go#L627-L630

### PR DESCRIPTION
**Pull request by jobec**
Fixes #242

### CODE REVIEW

File: exporter/redis.go [hunk 1]
```diff
@@ -624,11 +624,7 @@ func (e *Exporter) extractInfoMetrics(info, addr string, alias string, scrapes c
 			continue
 		}
 
-		split := strings.Split(line, ":")
-		if len(split) != 2 {
```
Comment (oliver006) [inline exporter/redis.go]: I'd keep the `len == 2` check. What if there's no `:` in the string?

**Comment by oliver006** [comment]
Thanks for the PR!

File: exporter/redis.go (hunk 1 above)
Comment (jobec) [reply exporter/redis.go]: Isn't that the check before it?

https://github.com/oliver006/redis_exporter/blob/e3f43cfe305f8d20d931de6eb84e32c664b7edd1/exporter/redis.go#L623-L625

Comment (oliver006) [reply exporter/redis.go]: You're very right, didn't see that in the diff. Good to go!

**Comment by jobec** [comment]
A new release would be great ;-)

**Comment by oliver006** [comment]
Coming right up!

**Comment by oliver006** [comment]
Released as https://github.com/oliver006/redis_exporter/releases/tag/v0.31.0

**Comment by jobec** [comment]
Could it be you forgot the binaries?

**Pull request merged**